src/Sulu/Bundle/WebsocketBundle/.gitignore

```diff
@@ -0,0 +1,15 @@
+/Tests/app/cache/*
+/Tests/app/logs/*
+/node_modules/
+/vendor/
+/composer.phar
+/composer.lock
+/.sass-cache/
+*~
+
+# IDEs
+.idea/*
+*.iml
+
+# temp files
+TEMP_*.php
```

Comment:
Then it is probably not required here? And it would be nice to know why this is in the content bundle as well.